theme/templates/includes/author-preview.html

```diff
@@ -1,85 +1,92 @@
-                            <th>HydroShare Profile: </th>
-                            <td><a data-profileurl href="#">View Profile</a></td>
-                        </tr>
-                        <tr>
-                            <td class="text-center"><i class="fa fa-building" aria-hidden="true"></i></td>
-                            <th>Organization:</th>
-                            <td data-organization></td>
-                        </tr>
-                        <tr>
-                            <td class="text-center"><i class="fa fa-envelope" aria-hidden="true"></i></td>
-                            <th>Email:</th>
-                            <td data-email></td>
-                        </tr>
-                        <tr>
-                            <td class="text-center"><i class="fa fa-map-marker" aria-hidden="true"></i></td>
-                            <th>Address:</th>
-                            <td data-address></td>
-                        </tr>
-                        <tr>
-                            <td class="text-center"><i class="fa fa-phone-square" aria-hidden="true"></i></td>
-                            <th>Phone:</th>
-                            <td data-phone></td>
-                        </tr>
-                        <tr>
-                            <td class="text-center"><i class="fa fa-globe" aria-hidden="true"></i></td>
-                            <th>Homepage:</th>
-                            <td><a href="#" target="_blank" data-homepage></a></td>
-                        </tr>
-                        <tr>
-                            <td class="text-center"><i class="fa fa-external-link-square" aria-hidden="true"></i></td>
-                            <th>Author Identifiers:</th>
-                            <td>
-                                <div class="external-profiles">
-                                    <a href="" data-googlescholarid class="identifier-icon" target="_blank" data-toggle="tooltip" data-placement="bottom"
-                                       title="Google Scholar">
-                                        <i class="ai ai-google-scholar-square hover-shadow"></i>
-                                    </a>
-                                    <a href="" data-orcid class="identifier-icon" target="_blank" data-toggle="tooltip" data-placement="bottom"
-                                       title="ORCID">
-                                        <i class="ai ai-orcid hover-shadow"></i>
-                                    </a>
-                                    <a href="" data-researchgateid class="identifier-icon" target="_blank" data-toggle="tooltip" data-placement="bottom"
-                                       title="ResearchGate">
-                                        <i class="ai ai-researchgate-square hover-shadow"></i>
+<script id="author-preview-modal-template" type="text/x-template">
```

Comment:
does it make sense to have the modal in a separate file or is that too extreme? I've been trying to break up colossal / run-on files and make top-level templates more readable. Downside, is tracing involves opening all the separate files... we should start to agree on some instincts of number of lines that warrants breaking things apart (1,000? 2,000?)

Reply:
Indeed. We have made progress in separating the code a little better. We used to have these modals inside genericresource.html, and we moved it into modals.html not long ago. We would need to discuss if it is worth it to separate this HTML even further into files for each element.